cli/pipeline/pipeline_definition.go

```diff
@@ -13,6 +13,7 @@ type PipelineDefinitionStep struct {
 	Version string `yaml:"version" json:"version"`
 	Branch  string `yaml:"branch" json:"branch"`
 	Image   string `yaml:"image" json:"image"`
+	Subdir  bool   `yaml:"subdir" json:"subdir"`
```

Comment:
Perhaps this should be implemented globally for the entire `yaml` in `PipelineDefinition`. Thoughts appreciated!

Reply:
I think it's best to implement it per-input, as that will give us the ability to migrate things over incrementally. I think you've got it in the correct place. :+1: